source/includes/serverless-compatibility.rst

```diff
@@ -0,0 +1,6 @@
+.. note::
+
+    To connect to Atlas Serverless, see the :atlas:`Serverless
```

Comment:
comment: Perhaps saying "For information about connecting to Atlas Serverless..." might be a bit better.